hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropBase.java

```diff
@@ -225,57 +208,7 @@ protected static SpecOperation[] setUpTokensWithCustomFees(long tokenTotal, long
                         .withCustom(
                                 royaltyFeeWithFallback(1, 2, fixedHbarFeeInheritingRoyaltyCollector(1), HTS_COLLECTOR)),
                 tokenAssociate(HTS_COLLECTOR, NFT_WITH_ROYALTY_FEE),
-                mintToken(NFT_WITH_ROYALTY_FEE, List.of(ByteStringUtils.wrapUnsafely("meta1".getBytes()))),
```

Comment:
Here and at `TokenAirdropTest` is there a merge issue here?  (Branch not up-to-date?). I know token airdrops are still being worked on so it could happen.  (I don't see a reason for such large changes.). (Possibly affects other suites as well?). (If so, should it be fixed sooner rather than later?)

Reply:
I think it is merge issue David. Fixed it. Thanks